packages/scene-composer/package.json

```diff
@@ -161,6 +161,7 @@
     "path-browserify": "1.0.1",
```

Comment:
This CR is showing 7 commits, however, our policy is that only one commit goes in per review. To resolve this, please do the following from your local machine:

* git reset HEAD~7
* add all files to tracking (ex: git add *)
* git commit -m 'YOUR_MESSAGE_HERE`
* git push --force

Reply:
Saw this too late, tried an interactive rebase and accidentally included all the commits that I tried to resolve merge conflicts with earlier🤦‍♂️. Trying to fix now

Reply:
Fixed :)